package.json

```diff
@@ -44,11 +44,12 @@
     "nyc": "^11.4.1",
     "pm2": "^2.9.1",
     "sequelize-cli": "^3.2.0",
-    "sinon": "^4.1.3"
+    "sinon": "^4.1.3",
+    "supports-color": "^5.1.0"
   },
   "dependencies": {
     "botanio": "0.0.6",
-    "debug": "^3.1.0",
+    "debug": "git+https://github.com/visionmedia/debug.git",
```

Comment:
Because https://github.com/visionmedia/debug/commit/285dfe10a5c06d4a86176b54bef2d7591eedaf40 not published to npm